doc/conf.py

```diff
@@ -25,23 +25,12 @@
 
 import sys
 import os
-from mock import Mock as MagicMock
-
-class Mock(MagicMock):
-    @classmethod
-    def __getattr__(cls, name):
-            return MagicMock()
-
-#MOCK_MODULES = ['pandas', 'pyOpenSSL']
-MOCK_MODULES = []
-sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
-
```

Comment:
I think for some reason we needed this, since the doc builds would have thrown errors.
Is it not necessary anymore?

Reply:
Hm, I can imagine it was necessary to insert mock modules at some point -- but since the ``MOCK_MODULES = ['pandas', 'pyOpenSSL']`` is currently commented out, I believe this code doesn't do anything. At least for me, the docs build locally :)

Reply:
+1